PolyEngine/Engine/Src/Imgui/ImguiWorldComponent.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <Defines.hpp>
+#include <ECS/ComponentBase.hpp>
+
+namespace Poly
+{
+	class ImguiResource;
+
+	class ENGINE_DLLEXPORT ImguiWorldComponent : public ComponentBase
```

Comment:
There really is no need for the component. This can be safely put into system, as it does not hold state relevant to the game.

Reply:
Done